source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -114,6 +114,24 @@ through results rather than returning all documents at once.
    :start-after: start fetchAll cursor example
    :end-before: end fetchAll cursor example
 
+Return Search Indexes of Collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using a v7.0 and later {+mdb-server+} cluster, you can use the new `listSearchIndex()
+<https://mongodb.github.io/node-mongodb-native/Next/classes/Collection.html#listSearchIndexes>`__
+to return a cursor with all the search indexes for the current
+collection. ``listSearchIndex()`` has an optional string parameter, ``name``, to
```

Comment:
S: Don't start sentences with monospace text if possible. Introduce methods as The ... method or types as the ... type